CONTRIBUTING.md

```diff
@@ -66,22 +39,118 @@ If you've found a bug that is not on the board, [follow these steps](#found-a-bu
 
 > _Updating to the latest releases is recommended_.
 
-### Getting Started
+### Forking The Project
 
-Note: If this is your first time working with a node-gyp dependent module,
-please follow the [node-gyp installation
-guide](https://github.com/nodejs/node-gyp#installation) to ensure a working npm
-build.
+#### Setting Up Your System
 
-The easiest way to get started is to clone the repository:
+1. Install [Git](https://git-scm.com/) or your favourite Git client.
+2. (Optional) [Setup an SSH Key](https://help.github.com/articles/generating-ssh-keys/) for GitHub.
+3. Create a parent projects directory on your system. For this guide, it will be assumed that it is `/mean/`
 
-```bash
-# Get the latest snapshot
-git clone --depth=1 https://github.com/freecodecamp/freecodecamp.git freecodecamp
+#### Forking FreeCodeCamp
+
+1. Go to the top level Free Code Camp repository: <https://github.com/FreeCodeCamp/freecodecamp>
+2. Click the "Fork" Button in the upper right hand corner of the interface ([More Details Here](https://help.github.com/articles/fork-a-repo/))
+3. After the repository has been forked, you will be taken to your copy of the FCC repo at `yourUsername/freecodecamp`
+
+#### Cloning Your Fork
+
+1. Open a Terminal / Command Line / Bash Shell in your projects directory (_IE: `/mean/`_)
```

Comment:
I think `IE` should be `i.e.` and our sample directory should be `/yourprojectdirectory/` instead of `/mean/` but this is opinion.